### PR TITLE
Update playbooks.md

### PR DESCRIPTION
**Pull request by DanielpFR**
Hi,
Please find few optimizations:
1- Changing the title, as Playbook is often used for Sentinel and may be confusing here (feedback from the MDI Bootcamp)
2 - Enabling a new attack simulation regarding DNS (it was there but not shown...)
3 - Adding additional information regarding the Account Reconnaissance
Thanks!
Danielp @Microsoft

### CODE REVIEW

**Comment by PRMerger9** [comment]
@DanielpFR : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by dcurwin** [comment]
Docs Build status updates of commit _[029b779](https://github.com/DanielpFR/ATADocs/commits/029b779064ee1d4ae96364b3d360c25195bb13bf)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[ATPDocs/playbooks.md](https://github.com/DanielpFR/ATADocs/blob/patch-2/ATPDocs/playbooks.md) | :warning:Warning | | [Details](#user-content-eef52d6972a269ab0e5fe32c91ac4d43-7565c277)

<a id="eef52d6972a269ab0e5fe32c91ac4d43-7565c277"></a>
### [ATPDocs/playbooks.md](https://github.com/DanielpFR/ATADocs/blob/patch-2/ATPDocs/playbooks.md)
 - **Line 2, Column 1**: **[Warning: title-missing - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/title-missing?branch=main)]** `````Missing required attribute: 'title'. Add a title string to show in search engine results.`````

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/5f256b2b-5ee0-2c08-3726-8de60259da9d/202208110943500949-559/BuildReport?accessString=b9e0b6b85ac6aa955968b2c8d97e258825129f0b0522a4a89704624fab5c3157).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/5f256b2b-5ee0-2c08-3726-8de60259da9d?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by dcurwin** [comment]
@DanielpFR - Thanks for your contribution. We'll review it and get back to you.

**Comment by Jak-MS** [comment]
IMPORTANT: When this content is ready to merge, the reviewer must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: ATPDocs/playbooks.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
 ---
-title: Microsoft Defender for Identity security lab playbook
-description: Learn how to simulate threats in your environment using the Microsoft Defender for Identity security lab playbooks.
+title: # Attack simulations for Microsoft Defender for Identity (MDI)
```
Comment (dcurwin) [inline ATPDocs/playbooks.md:2]:
```suggestion
title: Attack simulations for Microsoft Defender for Identity 
```

File: ATPDocs/playbooks.md [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
-title: Microsoft Defender for Identity security lab playbook
-description: Learn how to simulate threats in your environment using the Microsoft Defender for Identity security lab playbooks.
+title: # Attack simulations for Microsoft Defender for Identity (MDI)  
+description: Learn how to simulate threats in your environment using the Microsoft Defender for Identity security lab attack simulations.
 ms.date: 07/17/2022
 ms.topic: how-to
 ---
 
-# Playbook samples for Microsoft Defender for Identity
+# Attack simulations for Microsoft Defender for Identity (MDI)
```
Comment (dcurwin) [inline ATPDocs/playbooks.md:8]:
```suggestion
# Attack simulations for Microsoft Defender for Identity
```

**Comment by dcurwin** [comment]
Docs Build status updates of commit _[dd9f3e1](https://github.com/DanielpFR/ATADocs/commits/dd9f3e15d50e989dd071d0e6081a52af8d3fe941)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[ATPDocs/playbooks.md](https://github.com/DanielpFR/ATADocs/blob/patch-2/ATPDocs/playbooks.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/5f256b2b-5ee0-2c08-3726-8de60259da9d/202208221419289234-559/BuildReport?accessString=a1ec586ebf7110db2acf602f713eb56b641fffb34f62939e33873976695532b1).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/5f256b2b-5ee0-2c08-3726-8de60259da9d?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by dcurwin** [comment]
Docs Build status updates of commit _[dd9f3e1](https://github.com/DanielpFR/ATADocs/commits/dd9f3e15d50e989dd071d0e6081a52af8d3fe941)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[ATPDocs/playbooks.md](https://github.com/DanielpFR/ATADocs/blob/patch-2/ATPDocs/playbooks.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/5f256b2b-5ee0-2c08-3726-8de60259da9d/202208221423225696-559/BuildReport?accessString=9b2318945f99be1aea923f77965dcbaa69687b886bb3a561c27a217ef24a6230).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/5f256b2b-5ee0-2c08-3726-8de60259da9d?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by dcurwin** [comment]
@DanielpFR - Thanks for your contribution. I'm going to go ahead and publish it.

**Pull request merged**